sites/website/static/CNAME

```diff
@@ -1 +0,0 @@
-www.fast.design
```

Comment:
Was this left over for GH pages and not needed anymore?

Reply:
Correct, this was a requirement of GH Pages.